README.md

```diff
@@ -1,2 +1,151 @@
+
+**[React capstone project - Metrics webapp]** is...
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+>  Built with React, Redux and CSS, and linter configuration setup.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React JS</a></li>
+    <li><a href="https://redux.js.org/">Redux</a></li>
+    <li><a href="https://www.w3schools.com/w3css/defaulT.asp">CSS</a></li>
+    <li><a href="https://www.w3schools.com/js/default.asp">JavaScript</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+> Key features of the application.
+
+- **[Use Gitflow]**
+- **[Single page application]**
+- **[Category filtering]**
+- **[Show details page]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://react-capstone-project-metrics-webapp.onrender.com)
+- [Video Demo Link](https://www.loom.com/share/be5ac50475594070a3e202e5c1ab2916)
+- [Source Link](https://github.com/lenkon/react-capstone-project-metrics-webapp)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- node js
+- NPM (Node Package Manager)
+
+### Setup
+
+Clone this repository to your desired folder:
+
+``` 
+  $ git clone https://github.com/lenkon/react-capstone-project-metrics-webapp
+
+  $ cd bookstore
+```
+### Install
+
+Install all project dependencies by running the command below:
+
+``` 
+  $ npm install
+```
+### Usage
+
+Start the application server
+``` 
+  $ npm start
+```
+### Test
+Run the test using the command
+``` 
+  $ npm test
+```
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **lenkon**
+
+- GitHub: [@lenkon](https://github.com/lenkon)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Features to be added.
+
+- [ ] **[Add more projects]**
+- [ ] **[Add milestones and achievements]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please rate and give a star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for sharing the tools and requirements, and TMDB for the APIs used.
```

Comment:
- [ ] Kindly give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).